workspaces/rbac/plugins/rbac-node/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@backstage-community/plugin-rbac-node",
+  "description": "Node.js library for the rbac plugin",
+  "version": "1.8.0",
```

Comment:
latest version of janus version is 1.8.1 https://www.npmjs.com/package/@janus-idp/backstage-plugin-rbac-node